hook.php

```diff
@@ -61,7 +61,10 @@ function plugin_flyvemdm_uninstall() {
  * Also force inclusion of this file
  */
 function plugin_flyvemdm_postinit() {
-
+   $plugin = new Plugin();
+   if ($plugin->isActivated('orion')) {
+      PluginOrionReport::registerItemtype(PluginFlyvemdmPackage::class);
+   }
```

Comment:
this tells Orion that the given itemtype is linked to him. Orion will add a hook to remove entries in its tables when (persisted) instances of the itemtype are purged.